src/schema/v2/alerts.ts

```diff
@@ -66,9 +66,17 @@ const AlertType = new GraphQLObjectType<
 })
 
 const AlertsEdgeFields = {
-  totalUserSearchCriteriaCount: {
-    type: GraphQLInt,
-    resolve: ({ count_30d }) => count_30d,
+  counts: {
```

Comment:
Lets either have everything live under counts on the edge, or nothing live under counts on the edge. Counts seems better tho